.github/ISSUE_TEMPLATE/meeting.md

```diff
@@ -0,0 +1,33 @@
+---
+name: Community Meeting Announcement
+about: Announce the schedule, agenda, and participation details for Jeandle project community meetings
+title: "Community Meeting Announcement: YYYY-MM-DD"
+labels: meeting, community
+assignees: ''
+---
+
+## 🗓️ Meeting Schedule
+
+- Date: Please fill in the meeting date (e.g., 2025-09-12)
+- Time: Please fill in the meeting time (e.g., 20:00-21:00)
+- Format: Online meeting (please specify if asynchronous discussion is needed)
+- Meeting Link: To be added
+
+## 📋 Meeting Agenda
+
+1. Current unfixed issues and progress synchronization
+2. Community feedback and discussion
+3. Next phase planning and action
+
+Please leave comments below if you have additional agenda items.
+
+## 👥 How to Participate
+
+- Any community member (including contributors, Committers, Maintainers, etc.) may freely participate.
+- If unable to attend, please provide feedback in the comments section, and Maintainers will discuss it on your behalf during the meeting.
```

Comment:
All members may freely participate. So I think this line can be removed. Or we need to list who must attend.